modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,213 @@
+For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and mount that topic to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up system resources taken up by the topic, or migrate a topic to a different cluster.
```

Comment:
'detach segment data' is implementation detail.  

Something like:

'you can unmount a topic to safely detach it from a cluster while keeping the topic's data in the cluster's cloud storage bucket/container'